samples/scenarios/jdbc_kerberos/README.md

```diff
@@ -0,0 +1,152 @@
+  <selection-group>
+    <option value="auth-integrated" label="Integrated" />
+    <!-- other auth options -->
+  </selection-group>
+</field>
+```
+
+connectionProperties.js
+```
+    props["jaasApplicationName"] = "com.sun.security.jgss.krb5.initiate";
+    props["gsslib"] = "gssapi";	 
+    props["jaasLogin"] = "false";  
+```
+
+Other Configuration:
+* Only for postgres jdbc plugin on Mac - add this property `Settings.DisableNativeGSS` as a `Boolean/YES` to the Tableau plist on Mac at `/Library/Preferences/com.tableau.Tableau-<version>.plist` as described below. Here version is Tableau Desktop major version, for example, `com.tableau.Tableau-2020.4.plist`.
+```
+// Note - the below command requires root privileges hence sudo is needed.
+sudo defaults write /Library/Preferences/com.tableau.Tableau-<version>.plist Settings.DisableNativeGSS -bool YES
+```
+* Make sure a krb5.conf file is present at `/etc/krb5.conf`(/etc is a private directory, requires root privileges). Check for the existence of either of the following two files `/etc/krb5.conf` or `/Library/Preferences/edu.mit.Kerberos`. If the second file (edu.mit.Kerberos) is present it needs to be backed up and deleted.
+* Before connecting, run `klist` to verify kerberos TGT for the user principal is present in the kerberos ticket cache. If not, run `kinit user@REALM` before opening Tableau Desktop.  
+
+
+## <a id="server"/> Kerberos RunAs and Delegation on Server
+When a user logs into Tableau desktop using Integrated authentication, there are the two publish options available for Kerberos authentication, namely, "Server run-as" and "Viewer Credentials/Kerberos Delegation".
+
+![Image](images/PublishKerberosAuthOptions.png)
+
+### Server RunAs authentication
+You can configure Tableau Server to use a Kerberos service account to access a database. In this scenario, Tableau Server connects to databases with a service account, also referred to as a "RunAs account". 
+For Server RunAs authentication, users need to provide a user principal name and a keytab file path. Tableau Server uses the keytab to login to kerberos KDC using JAAS and the TGT thus obtained is used by jdbc driver for making a database connection.
+For configuring a Linux Server see this Tableau [article](https://help.tableau.com/current/server-linux/en-us/kerberos_runas_linux.htm). For configuring a Windows Server see this Tableau [article](https://help.tableau.com/current/server/en-us/kerberos_runas_jdbc.htm). 
+These are the plugin code changes required to support Server RunAs authentication on Tableau Server.
+
+connectionProperties.js
+```
+    function isEmpty(str) {
+        return (!str || 0 === str.length); 
+    }
+
+    if (attr[connectionHelper.attributeAuthentication] == "auth-integrated") {
+        // if attributeTableauServerUser is non-empty, it means the connector plugin is currently being accessed in a tableau server environment
+        var serverUser = attr[connectionHelper.attributeTableauServerUser];
```

Comment:
This is the run as user here right? Might want to add a comment to explain that. Also this example is the same as below, maybe just use one code example with the comment that explains that in the run-as case the tableauServerUser which is passed down is the runAsUser.